### DISCUSSION TITLE
Saving opened files/windows list to text file along with path

### DISCUSSION THREAD

**Discussion by santoshicha** [unixwork/xnedit#121] [Ideas]
Does XNEDIT has the feature to save list of files opened in a window to textfile along with absolute path?
This is very much useful to switch the X session.

Thank you.

**Comment by unixwork** [comment]
This is a highly requested feature that is already in development, but it is not ready yet. There is also the issue #104 about this topic.

Currently, the code for this is in the `session` branch. It is a while since I worked on this and I don't remember how well everything works, but the basic session save/restore functionality should work.
